safe/impact_function/test/test_postprocessors.py

```diff
@@ -38,6 +42,11 @@
     post_processor_youth,
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->